dc_utils/templates/404.html

```diff
@@ -3,26 +3,28 @@
     <head>
         <meta charset="utf-8">
         <title>
-            Page Not found
+            Page not found
         </title>
+        <link rel="stylesheet" href="http://dc-shared-frontend-assets.s3-website.eu-west-2.amazonaws.com/styles.css">
     </head>
     <body class="ds-width-full">
         <div class="ds-page">
             <p><a class="ds-skip-link" href="#main">skip to content</a></p>
-            <header role="banner">
+            <header class="ds-header">
                 <div class="container">
-                    <a href="/">
-                        <img src="https://dc-shared-frontend-assets.s3.eu-west-2.amazonaws.com/images/logo_icon.svg" alt="{% if site_title %}{{ site_title }} logo{% endif %}" width="72">
+                    <a class="ds-logo" href="/">
+                        <img src="https://dc-shared-frontend-assets.s3.eu-west-2.amazonaws.com/images/logo_icon.svg" alt="Democracy Club logo" width="72">
+                        <span>{{ SITE_TITLE|default:"democracy<br>club" }}{% block language_code %}{% endblock language_code %}</span>
```

Comment:
What would we envisage putting in the `language_code` block?

Reply:
![image](https://github.com/user-attachments/assets/e66e162a-4d9a-460b-a040-67c6bff27356)

![image](https://github.com/user-attachments/assets/c8f66e73-58ad-4210-bbea-7cf1e23c9c67)

Reply:
OK, ta. To me `language_code` is the code itself (i.e: 'cy' or 'en-GB' or whatever), not the content in that language, but I guess we are already some distance into this naming convention now.

Reply:
I agree it's confusing! But yeah, we're using it in at least one place so I wanted to keep it as is in this PR